File: demo-projects/Typescript/hello-world.ts
class Greeter {
	greeting: string;
	constructor(message: string) {
		this.greeting = message;
	}

	SayHello() {
		console.log(this.greeting + " world");
	}
}

let greeter = new Greeter("Hallo");
greeter.SayHello()

File: demo-projects/Go/hello-world.go
package main

import "fmt"

type Greeter struct {
	Greeting string
}

func (g Greeter) SayHello() {
	fmt.Println(g.Greeting, "world")
}

func main() {
	greeter := Greeter{Greeting: "Ola"}
	greeter.SayHello()
}
